src/Knp/Rad/User/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Knp\Rad\User\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('knp_rad_user');
+
+        $rootNode
+            ->children()
+                ->scalarNode('doctrine_driver')
```

Comment:
Maybe you can use 
```php
        $rootNode
        ->children()
            ->enumNode('doctrine_driver')
                ->values(['orm', 'mongodb', 'couchdb'])
                ->defaultValue('orm')
            ->end()
        ->end();
```
Instead of the validation.